src/QueryM.purs

```diff
@@ -848,7 +789,42 @@ mkOgmiosRequest jsonWspCall getLs inp = do
       _ <- respLs.addMessageListener id
         ( \result -> do
             respLs.removeMessageListener id
-            allowError cont $ result
+            cont $ lmap dispatchErrorToError result
+        )
+      respLs.addRequest id sBody
+      _wsSend ws (logString config.logLevel Debug) sBody
+      pure $ Canceler $ \err -> do
+        liftEffect $ respLs.removeMessageListener id
+        liftEffect $ throwError $ err
+  liftAff $ makeAff $ affFunc
+
+-- TODO after ogmios-datum-cache implements reflection this could be generalized to make request for the cache as well
```

Comment:
I thought using ODC's new reflection mechanism was the rationale for this PR?

Reply:
Yep, a copy-paste fail on my side.